roles/docker/defaults/main.yml

```diff
@@ -10,3 +10,33 @@ docker_repo_info:
   repos:
 
 docker_dns_servers_strict: yes
+
+docker_container_storage_setup: false
```

Comment:
It's not atomic specific, it's just that right now the feature is not tested on debian family. It would be trivial to extend this functionality. I did some basic testing on debian, but couldn't get devicemapper storage (the default) to work, so I've made this RH specific for now.